service/key/key.go

```diff
@@ -203,7 +204,9 @@ func APIUrl(obj interface{}) string {
 	case *v1.Service:
 		return fmt.Sprintf("%s:443", v.Spec.ClusterIP)
 	case *capiv1alpha3.Cluster: // Support CAPI Clusters
-		return fmt.Sprintf("%s:%d", v.Spec.ControlPlaneEndpoint.Host, v.Spec.ControlPlaneEndpoint.Port)
+		host := strings.TrimPrefix(v.Spec.ControlPlaneEndpoint.Host, "https://")
```

Comment:
Why did this changed?

Reply:
Because eks CAPI clusters adds the https at the beginning :(

Reply:
Alright